policyengine_us/parameters/gov/states/ca/tax/income/credits/foster_youth/base.yaml

```diff
@@ -0,0 +1,30 @@
+description: California Foster Youth Tax Credit is limited to person between 18 to 25 and has a maximum amount of 1083
+brackets:  
+  - threshold:  
+      2022-01-01: 0  
+    amount:  
+      2022-01-01: 0  
+  - threshold:  
+      2022-01-01: 18  
+    amount:  
+      2022-01-01: 1_176
+  - threshold:  
+      2022-01-01: 25  
+    amount:  
+      2022-01-01: 0 
+metadata:
+  threshold_unit: year
+  type: single_amount
+  amount_unit: currency-USD
+  period: year
+  label: California foster youth tax credit base  
+  reference:
+    - title: California Foster Youth Tax Credit
+      href: https://www.ftb.ca.gov/file/personal/credits/foster-youth-tax-credit.html
+    - title: 2022 California Earned Income Tax Credit Form
+      href: https://www.ftb.ca.gov/forms/2022/2022-3514.pdf#page=4
+    #- title: 2021 California Earned Income Tax Credit Form
+      #href: https://www.ftb.ca.gov/forms/2021/2021-3514.pdf#page=3
```

Comment:
delete if not necessary and add a comment that the credit started in 2022 if the legal code states so

Reply:
all parameter files